scripts/actions/abilities/pets/perfect_defense.lua

```diff
@@ -24,6 +24,8 @@ abilityObject.onPetAbility = function(target, pet, skill, master)
     target:addStatusEffect(xi.effect.PERFECT_DEFENSE, power, 3, duration)
     skill:setMsg(xi.msg.basic.SKILL_GAIN_EFFECT)
```

Comment:
Check reraise_ii Cait South lua.

You need to use the job ability messages now that you're using pet skills

Reply:
This should be fixed now.